js/src/jit/BaselineCacheIRCompiler.cpp

```diff
@@ -2688,6 +2741,34 @@ ICAttachResult js::jit::AttachBaselineCacheIRStub(
   return ICAttachResult::Attached;
 }
 
+#ifdef ENABLE_JS_AOT_ICS
+
+#ifndef ENABLE_PORTABLE_BASELINE_INTERP
+// The AOT loading of ICs doesn't work (yet) in modes with a native
+// JIT enabled because compilation tries to access state that doesn't
+// exist yet (trampolines?) when we create the JitZone.
+#error AOT ICs are only supported (for now) in PBL builds.
+#endif
+
+void js::jit::FillAOTICs(JSContext* cx, JitZone* zone) {
+  if (JitOptions.enableAOTICs) {
+    for (auto& stub : GetAOTStubs()) {
```

Comment:
I can't see an definition or implementation of this function anywhere, are we missing the file named `CacheIRAOT.h`?

Reply:
Added, sorry!